Update hip-869.md (#1019)
hashgraph/hedera-improvement-proposal: HIP/hip-869.md

This HIP proposes the introduction of a new NodeService API that enables a node operator to create, delete, and update nodes. All of these transactions must be signed by the Hedera Council.

```protobuf
service AddressBookService {
    /**

>      * A transaction to create a new consensus node in the network.
>      * address book.

Reviewer: ```suggestion
     * A transaction to create a new consensus node in the network
     * address book.
```

Author: Updated.

     * <p>
     * This transaction, once complete, SHALL add a new consensus node to the
     * network state.<br/>
     * The new consensus node SHALL remain in state, but SHALL NOT participate
     * in network consensus until the network updates the network configuration.
     * <p>
     * Hedera governing council authorization is REQUIRED for this transaction.
     */
    rpc createNode (proto.Transaction) returns (proto.TransactionResponse);

    /**
     * A transaction to remove a consensus node from the network address
     * book.
     * <p>
     * This transaction, once complete, SHALL remove the identified consensus
     * node from the network state.
     * <p>
     * Hedera governing council authorization is REQUIRED for this transaction.
     */
    rpc deleteNode (proto.Transaction) returns (proto.TransactionResponse);

    /**
     * A transaction to update an existing consensus node from the network
     * address book.
     * <p>
     * This transaction, once complete, SHALL modify the identified consensus
     * node state as requested.
     * <p>

>      * This transaction MAY be authorized by either the node operator OR the
>      * Hedera governing council.

Reviewer: This changed after we first wrote the text, it's only node operator (`admin_key`) now.
```suggestion
     * This transaction MUST be authorized by the node operator.
```

Author: Updated.
